src/screens/GroupSetting/index.js

```diff
@@ -105,7 +104,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600],
     fontSize: 14,
     lineHeight: 16.94,
-    color: colors.darkBlue,
+    color: COLORS.anon_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been updated to use the new `COLORS` object. This change aligns with the PR's objective of standardizing color usage across the application. The new color being used (`COLORS.anon_primary`) should be verified to ensure it matches the intended design.


<!-- This is an auto-generated comment by OSS CodeRabbit -->